src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+	</div>
+);
+
+const update1801: ReactNode = (
+	<div className="paragraph">
+		<ul>
+			<li>
+				<p>KnownId - Removed: <code className="small-code-sample">UnitOfMeasureLitersPer100KilometersId</code>. Diagnostics associated with this unit of measure now use <code className="small-code-sample">UnitOfMeasureKilometersPerLiterId</code>. This will not affect any previously recorded data.</p>
+			</li>
+			<li>
+				<p>KnownId - Added: <code className="small-code-sample">DiagnosticGpsLogReasonId</code>, <code className="small-code-sample">DiagnosticEngineRoadSpeedId</code></p>
+			</li>
+			<li>
+				<p>ConditionType - Added: <code className="small-code-sample">DVIRDefect</code> - Currently works with Devices not Trailers</p>
+			</li>
+			<li>
+				<p>SecurityIdentifier - Added: <code className="small-code-sample">DriverIdentificationClearance</code>, <code className="small-code-sample">AccelerometerDataClearance</code>, <code className="small-code-sample">ServicePlansClearance</code>, <code className="small-code-sample">AuxiliaryClearance</code>, <code className="small-code-sample">EngineStatusDataClearance</code>, <code className="small-code-sample">ResellerControlClearance</code>, <code className="small-code-sample">GoTalkClearance</code>, <code className="small-code-sample">StatusGroupsClearance</code>, <code className="small-code-sample">ProductGuideClearance</code>, <code className="small-code-sample">FeaturePreviewClearance</code>, <code className="small-code-sample">NewsNotificationsClearance</code>, <code className="small-code-sample">ManageAddinsClearance</code>, <code className="small-code-sample">DeviceCurrentStatusClearance</code></p>
+			</li>
+		</ul>
+		<h2>Notice</h2>
+		<p>In early 2018 the following legacy properties will be removed:</p>
+		<p><b>Authenticate</b>: <code className="small-code-sample">userLogin</code> parameter. This was kept around for compatibility with legacy (5.6.* and lower) integrations. It has not been publicly exposed or documented since version 5.6 of MyGeotab. It is planned to be removed as a valid parameter in version 5.7.1803. The <code className="small-code-sample">userName</code> parameter is the standard supported property that should be used.</p>
+		<p><b>LoginResult</b>: <code className="small-code-sample">securityToken</code> property. LoginResult is the object returned by the Authenticate method. It’s property <code className="small-code-sample">securityToken</code> was kept around for compatibility with legacy (5.6.* and lower) integrations. It has not been publicly exposed or documented since version 5.6 of MyGeotab. It is planned to be removed as a valid parameter in version 5.7.1803. The property <code className="small-code-sample">credentials</code> is the standard supported property that shares the same value.</p>
```

Comment:
The character U+2019 "’" could be confused with the ASCII character U+0060 "`", which is more common in source code